scipy/interpolate/ndgriddata.py

```diff
@@ -191,6 +191,9 @@ def griddata(points, values, xi, method='linear', fill_value=np.nan,
 
     """
 
+    if type(values) != np.ndarray:
+          values = np.array(values)
```

Comment:
The idiomatic way of doing this would be `values = np.asarray(values)`. However, as noted in gh-9433, `griddata` is documented to accept `ndarray`, not `array_like`. No idea why not though.